File: ProjectPO_Java/src/PO_java/LifeSymulator/Plants/Grass.java
package PO_java.LifeSymulator.Plants;

import PO_java.LifeSymulator.Plant;
import PO_java.LifeSymulator.Tools.Cords;
import PO_java.LifeSymulator.World;

public class Grass extends Plant {
    public Grass(World world, Cords cords)
    {
        strength = 0;
        age = 0;
        sign = '"';
        name = "Grass";
        toKill = false;
        isDead = false;
        this.cords = cords;
        this.world = world;
    }
}

File: ProjectCppPO/Grass.cpp
#include "Grass.h"

Grass::Grass(World* world, Cords cords)
{
	features.strength = 0;
	features.age = 0;
	features.look = '"';
	features.name = "Grass";
	features.toKill = false;
	location = cords;
	this->world = world;
}
